ops/cli.py

```diff
@@ -0,0 +1,249 @@
+        code1 = subprocess.call(GCLOUD_SETUP, shell=True)
+
+        print(
+            dye_msg_with_color(
+                msg=f"=> Setting up Kubernetes cluster context", color="blue"
+            )
+        )
+        CTX = f"gke_broad-gotc-{environment}_us-central1-a_gotc-{environment}-shared-us-central1-a"
+        KUBE_CONTEXT = f"kubectl config use-context {CTX}"
+        code2 = subprocess.call(KUBE_CONTEXT, shell=True)
+
+        code3 = 0
+        if namespace:
+            KUBE_SET_NAMESPACE = f"kubectl config set-context $(kubectl config current-context) --namespace={namespace}"
+            code3 = subprocess.call(KUBE_SET_NAMESPACE, shell=True)
+            print(
+                dye_msg_with_color(
+                    msg=f"=> Set up namespace to {namespace}", color="green"
+                )
+            )
+        return any([code1, code2, code3])
+
+    @staticmethod
+    def _helm_deploy_wfl(values: str) -> int:
+        """Use Helm to deploy WFL to K8S."""
+        print(
+            dye_msg_with_color(
+                msg=f"=> Deploying to K8S cluster with Helm (this requires VPN)",
+                color="blue",
+            )
+        )
+        HELM_UPGRADE = f"helm upgrade wfl-k8s gotc-charts/wfl -f {values} --install"
+        print(
+            dye_msg_with_color(
+                msg=f"=> WFL is deployed, run `kubectl get pods` to see details",
+                color="green",
+            )
+        )
+        return subprocess.call(HELM_UPGRADE, shell=True)
+
+    @register
+    def deploy(self, arguments):
+        """Deploy WFL to Cloud GKE (VPN is required)"""
+        parser = argparse.ArgumentParser(description=f"{self.deploy.__doc__}")
+        parser.add_argument(
+            "-e",
+            "--environment",
+            dest="environment",
+            default="dev",
+            choices={"dev",},
+            help="Environment to deploy to",
+        )
+        parser.add_argument(
+            "-n",
+            "--namespace",
+            dest="namespace",
+            default="",
+            help="The namespace you want to use for deployemnt",
+        )
+        args = parser.parse_args(arguments)
+
+        CLI._is_available("helm", "kubectl", "gcloud", "git", "docker")
+        CLI._set_up_helm()
+        CLI._set_up_k8s(namespace=args.namespace)
+
+        with tempfile.TemporaryDirectory(
+            prefix=str(uuid.uuid4()), dir=Path("./")
+        ) as dir_name:
+            CLI._clone_config_repo(
+                dest=Path(f"./{dir_name}/gotc-deploy"), branch="master"
+            )
+
+            file_name = "wfl-values.yaml.ctmpl"
+            shutil.copy(
+                str(Path(f"./{dir_name}/gotc-deploy/deploy/gotc-dev/helm/{file_name}")),
+                Path(f"./{dir_name}/{file_name}"),
+            )
+            CLI._render_ctmpl(ctmpl_file=str(Path(f"./{dir_name}/{file_name}")))
```

Comment:
Why is the ctmpl file rendered in two places (the render command and in the deploy command)? Is the render command meant to be a way to review the values?

Reply:
I was thinking the `render` command could work as a utility command so we could use it in other situations, e.g. render the auth-proxy CTMPL files.